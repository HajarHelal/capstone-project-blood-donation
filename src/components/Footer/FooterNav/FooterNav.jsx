import React from "react";
import {
  BrowserRouter as Router,
  Switch,
  Route,
  Link
} from "react-router-dom";
import Home from '../../../containers/Home';
import About from '../../../containers/About';
import Instruction from '../../../containers/Instruction';
import Process from '../../../containers/Process';
import Search from '../../../containers/Search';
import Donate from '../../../containers/Donate';

function FooterNav(){

    const NavDesign ='px-2 py-2 lg:px-6  mt-4 md:mt-0 sm:mr-0 md:mr-2 lg:mr-4 text-sm  text-white tracking-wide inline-block border-b-2 border-transparent hover:border-white'
    return (
<Router>
<div classname="flex justify-center flex-wrap">
        <nav>
          <ul className=" text-white sm:flex justify-center flex-wrap ">
            <li className={NavDesign}>
              <Link to="/">Home</Link>
            </li>
            <li className={NavDesign}>
              <Link to="/about">About</Link>
            </li>
            <li className={NavDesign}>
              <Link to="instruction">Instructions</Link>
            </li>
            <li className={NavDesign}>
              <Link to="/process">Process</Link>
            </li>
            <li className={NavDesign}>
              <Link to="/search">Search</Link>
            </li>
            <li className={NavDesign}>
              <Link to="/donate">Donate</Link>
            </li>
          </ul>
        </nav>
          
        <Switch>
          <Route path="/about">
            <About />
          </Route>
          <Route path="/instruction">
            <Instruction/>
          </Route>
          <Route path="/process">
            <Process/>
          </Route>
          <Route path="/search">
            <Search/>
          </Route>
          <Route path="/donate">
            <Donate/>
          </Route>
          <Route path="/">
            <Home />
          </Route>
        </Switch>
        </div>

</Router>
      
    )
}
export default FooterNav