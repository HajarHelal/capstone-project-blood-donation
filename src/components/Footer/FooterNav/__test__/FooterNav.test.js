import React from 'react';
import renderer from 'react-test-renderer';
import FooterNav from '../FooterNav';

// tests will use to compare to.
it('renders correctly', () => {
  const tree = renderer.create(<FooterNav />).toJSON();
  expect(tree).toMatchSnapshot();
});