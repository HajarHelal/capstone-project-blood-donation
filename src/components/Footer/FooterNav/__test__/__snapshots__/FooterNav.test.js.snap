// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  classname="flex justify-center flex-wrap"
>
  <nav>
    <ul
      className=" text-white sm:flex justify-center flex-wrap "
    >
      <li
        className="px-2 py-2 lg:px-6  mt-4 md:mt-0 sm:mr-0 md:mr-2 lg:mr-4 text-sm  text-white tracking-wide inline-block border-b-2 border-transparent hover:border-white"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li
        className="px-2 py-2 lg:px-6  mt-4 md:mt-0 sm:mr-0 md:mr-2 lg:mr-4 text-sm  text-white tracking-wide inline-block border-b-2 border-transparent hover:border-white"
      >
        <a
          href="/about"
          onClick={[Function]}
        >
          About
        </a>
      </li>
      <li
        className="px-2 py-2 lg:px-6  mt-4 md:mt-0 sm:mr-0 md:mr-2 lg:mr-4 text-sm  text-white tracking-wide inline-block border-b-2 border-transparent hover:border-white"
      >
        <a
          href="/instruction"
          onClick={[Function]}
        >
          Instructions
        </a>
      </li>
      <li
        className="px-2 py-2 lg:px-6  mt-4 md:mt-0 sm:mr-0 md:mr-2 lg:mr-4 text-sm  text-white tracking-wide inline-block border-b-2 border-transparent hover:border-white"
      >
        <a
          href="/process"
          onClick={[Function]}
        >
          Process
        </a>
      </li>
      <li
        className="px-2 py-2 lg:px-6  mt-4 md:mt-0 sm:mr-0 md:mr-2 lg:mr-4 text-sm  text-white tracking-wide inline-block border-b-2 border-transparent hover:border-white"
      >
        <a
          href="/search"
          onClick={[Function]}
        >
          Search
        </a>
      </li>
      <li
        className="px-2 py-2 lg:px-6  mt-4 md:mt-0 sm:mr-0 md:mr-2 lg:mr-4 text-sm  text-white tracking-wide inline-block border-b-2 border-transparent hover:border-white"
      >
        <a
          href="/donate"
          onClick={[Function]}
        >
          Donate
        </a>
      </li>
    </ul>
  </nav>
  <div>
    Home
  </div>
</div>
`;
